include/dxc/dxctools.h

```diff
@@ -29,6 +29,16 @@ IDxcRewriter : public IUnknown {
                                                      _In_ UINT32 defineCount,
                                                      _COM_Outptr_ IDxcOperationResult **ppResult) = 0;
 
+  virtual HRESULT STDMETHODCALLTYPE RewriteUnchangedWithInclude(_In_ IDxcBlobEncoding *pSource,
+                                                     // Optional file name for pSource. Used in errors and include handlers.
+                                                     _In_opt_ LPCWSTR pSourceName,
+                                                     _In_count_(defineCount) DxcDefine *pDefines,
+                                                     _In_ UINT32 defineCount,
+                                                     // user-provided interface to handle #include directives (optional)
+                                                     _In_opt_ IDxcIncludeHandler *pIncludeHandler,
+                                                     bool  bSkipFunctionBody,
```

Comment:
bool should be BOOL (ABI). But maybe you can use UINT32 flags, too, in case we want some variants as well.